src/services/AuthenticationService.jsx

```diff
@@ -15,3 +15,7 @@ export const registerUser = (username, password) => {
     password2: password,
   });
 };
+
+export const getCurrentUser = () => {
+  return axios.get("/author/current");
```

Comment:
should we check for a status 200 here (like how we do in the PostService)? We didn't add a check for the other 2 functions in this file but we can do that later.

Reply:
It is checked in PrivateRoute when I use it. 
[ auth.getCurrentUser()
      .then(((response) => {
        if (response.status === 200) {...](url)